index.js

```diff
@@ -1,8 +1,6 @@
 /* jshint node: true */
```

Comment:
Is this file used? `Package.json` points at `lib/cli/index.js` as the main file, so I'm not sure if this one is used by anything.